Extension/src/LanguageServer/extension.ts

```diff
@@ -1430,18 +1430,27 @@ async function onCopilotHover(): Promise<void> {
 
     // Gather the content for the query from the client.
     const requestInfo = await copilotHoverProvider.getRequestInfo(hoverDocument, hoverPosition);
-    if (requestInfo.length === 0) {
+    for (const file of requestInfo.files) {
+        const fileUri = vscode.Uri.file(file);
+        if (await vscodelm.fileIsIgnored(fileUri, copilotHoverProvider.getCurrentHoverCancellationToken() ?? CancellationToken.None)) {
```

Comment:
Given that this is technically still a proposed API (even if it is expected to be stable), we should wrap the call to `fileIsIgnored()` in a try/catch.

Reply:
Added